README.md

```diff
@@ -40,7 +40,21 @@ For local development, we recommend to use the following setup which will enable
 pip install --upgrade pip
 git clone git@github.com:awslabs/syne-tune.git
 cd syne-tune
-pip install -e .[extra]
+pip install -e '.[extra]'
```

Comment:
Without quotes install did not work for me. See also here https://stackoverflow.com/questions/52228286/how-to-pip-install-with-wheel-specifying-extras .

Reply:
If we do this, we should also do it for all "pip install" instructions for uniformity.

Reply:
Added this to other places, and made sure instructions still work by running updated pip install commands.